leaflet-treering.js

```diff
@@ -127,13 +127,16 @@ function LTreering (viewer, basePath, options) {
       this.ioTools.bar.addTo(this.viewer);
       this.settings.bar.addTo(this.viewer);
       this.undoRedoBar.addTo(this.viewer);
+      //removes 'annotations' layer
+      map.removeLayer(this.annotationAsset.markerLayer);
     } else {
       this.popout.btn.addTo(this.viewer);
       this.viewData.btn.addTo(this.viewer);
       this.ioTools.bar.addTo(this.viewer);
       this.settings.bar.addTo(this.viewer);
-      //defaults overlay 'points' option to disabled
+      //defaults overlay 'points' & 'annotations' option to disabled
       map.removeLayer(this.visualAsset.markerLayer);
+      map.removeLayer(this.annotationAsset.markerLayer);
```

Comment:
if you're removing the annotation layer in both cases, move it outside the conditional